doc/visual-programming/source/widgets/visualize/radviz.rst

```diff
@@ -0,0 +1,94 @@
+Radviz
+======
+
+.. image:: icons/radviz.png
+
+Radviz vizualization with explorative data analysis and intelligent data
+visualization enhancements.
+
+Signals
+-------
+
+**Inputs**:
+
+-  **Data**
+
+   An input data set.
+
+-  **Data Subset**
+
+   A subset of instances from the input data set.
+
+**Outputs**:
+
+-  **Selected Data**
+
+   A subset of instances that the user manually selected from the
+   Radviz plot
+
+-  **Annotated Data**
+
+   Data with an additional column showing whether a point is selected. If more
+   than one group is selected then also the group name is written instead.
+
+-  **Components**
+
+   Radviz vectors
+
+
+
+Description
+-----------
+
+Radviz ([1]_) is a neat non-linear multi-dimensional visualization
+technique that can display data on three or more attributes in a 2-dimensional
+projection. The visualized attributes are presented as anchor points equally
+spaced around the perimeter of a unit circle. Data instances are shown as
+points inside the circle, with their positions determined by a metaphor from
+physics: each point is held in place with springs that are attached at the
+other end to the attribute anchors. The stiffness of each spring is
+proportional to the value of the corresponding attribute and the point ends up
+at the position where the spring forces are in equilibrium. Prior to
+visualization, attribute values are scaled to lie between 0 and 1. Data
+instances that are close to a set of feature anchors have higher values for
+these features than for the others.
+
+The snapshot shown below shows a Radviz widget with a visualization of the
+data set from functional genomics ([2]_). In this particular
+visualization the data instances are colored according to the corresponding
+class, and the visualization space is colored according to the computed class
+probability. Notice that the particular visualization very nicely separates
+the data instances of the different class, making the visualization interesting
+and potentially informative.
+
+.. image:: images/Radviz-Brown.png
+
+Just like all point-based visualizations, this widget includes tools for
+intelligent data visualization (VizRank, see [3]_) and interface for explorative
+data analysis - selection of data points in visualization. Just like in
+:ref:`Scatter Plot` widget, intelligent visualization can be used to find a set
+of attributes that would result in an interesting visualization. For now, this
+works only with class-labeled data set, where interesting visualizations are
+those that well separate data instances of different class. Radviz graph above
+is according to this definition an example of a very good visualization,
+while the one below - where we show an VizRank's interface (:obj:`VizRank` button in
```

Comment:
the one below is not.
The extra sentence can IMO be omitted, if not add it separately (otherwise hard to read).